packages/gimbal/src/vcs/comment/index.ts

```diff
@@ -1,25 +1,54 @@
 import EventEmitter from '@modus/gimbal-core/lib/event';
 import whichCI from '@/ci';
 import Config from '@/config';
-import { outputTable } from '@/output/markdown';
+import { createTable } from '@/output/cli';
+import { outputTable, tableConfig } from '@/output/markdown';
 import { Report, ReportItem } from '@/typings/command';
 import { CommandOptions } from '@/typings/utils/command';
-import { CommentStartEvent, CommentEndEvent } from '@/typings/vcs/comment';
+import {
+  CommentBuildStartEvent,
+  CommentBuildEndEvent,
+  CommentRenderTableStartEvent,
+  CommentRenderTableEndEvent,
+  CommentStartEvent,
+  CommentEndEvent,
+} from '@/typings/vcs/comment';
 
-const renderItem = (item: ReportItem, options: CommandOptions): string => {
-  if (!item.data) {
+const renderItem = async (reportItem: ReportItem, commandOptions: CommandOptions): Promise<string> => {
+  if (!reportItem.data) {
     return '';
   }
 
-  const numFailed = item.data.reduce(
+  const numFailed = reportItem.data.reduce(
     (num: number, dataItem: ReportItem): number => num + (dataItem.success ? 0 : 1),
     0,
   );
 
-  return `<details><summary>${item.label} (${numFailed} failure${numFailed === 1 ? '' : 's'})</summary>
+  const table = createTable(commandOptions, tableConfig);
+
+  const commentRenderTableStartEvent: CommentRenderTableStartEvent = {
+    commandOptions,
+    reportItem,
+    table,
+  };
+
+  await EventEmitter.fire(`vcs/comment/render/table/start`, commentRenderTableStartEvent);
```

Comment:
This event is key to this particular issue, this is what the last-value plugin will listen on to add the column.